apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -41,65 +41,67 @@ const MeterReading = new GQLListSchema('MeterReading', {
+            //         if (operation === 'create') {
+            //             const value = Number(resolvedData[fieldPath])
+            //             const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
+            //
+            //             if (lastMeterReading && lastMeterReading.value1 > value)
+            //                 addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
+            //         }
+            //     },
+            // },
         },
 
         value2: {
             schemaDoc: 'If the meter is multi-tariff, then the value of the second tariff is stored here',
             type: Decimal,
-            hooks: {
-                validateInput: async ({ context, operation, resolvedData, fieldPath, addFieldValidationError }) => {
-                    if (operation === 'create') {
-                        const value = Number(resolvedData[fieldPath])
-                        const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
-
-                        if (lastMeterReading && lastMeterReading.value2 > value)
-                            addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
-                    }
-                },
-            },
+            // hooks: {
+            //     validateInput: async ({ context, operation, resolvedData, fieldPath, addFieldValidationError }) => {
+            //         if (operation === 'create') {
+            //             const value = Number(resolvedData[fieldPath])
+            //             const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
+            //
+            //             if (lastMeterReading && lastMeterReading.value2 > value)
+            //                 addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
+            //         }
+            //     },
+            // },
         },
 
         value3: {
             schemaDoc: 'If the meter is multi-tariff, then the value of the third tariff is stored here',
             type: Decimal,
-            hooks: {
-                validateInput: async ({ context, operation, resolvedData, fieldPath, addFieldValidationError }) => {
-                    if (operation === 'create') {
-                        const value = Number(resolvedData[fieldPath])
-                        const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
-
-                        if (lastMeterReading && lastMeterReading.value3 > value)
-                            addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
-                    }
-                },
-            },
+            // hooks: {
```

Comment:
You can delete these lines if they are no longer needed?

Reply:
We will most likely come back to this. I thought it would be better to comment it out

Reply:
Please add TODO with task on either removing or restoring the commentary blocks :)

Reply:
Removed all comented lines from this pr in #823